sentry2csv/sentry2csv.py

```diff
@@ -175,4 +189,10 @@ def main():
         logger.setLevel(logging.WARNING)
     enrichments = extract_enrichment(args.enrich)
     loop = asyncio.get_event_loop()
-    loop.run_until_complete(export(args.token[0], args.organization[0], args.project[0], enrich=enrichments))
+    loop.run_until_complete(
+        export(args.token[0], args.organization[0], args.project[0], enrich=enrichments, host=args.host[0])
```

Comment:
If no host is provided, `args.host` is empty. We should probably specify a default value on the argument declaration.